index.js

```diff
@@ -106,25 +106,28 @@ module.exports = {
       },
 
       status                   : function () {
-        return {progress: self.progress(), db: ssb.status, sync: since() }
+        return {
+          progress: self.progress(),
+          db: ssb.status,
+          sync: since()
+        }
       },
 
       version                  : function () {
         return pkg.version
       },
 
       //temporary!
-      _flumeUse                :
-        function (name, flumeview) {
-          ssb.use(name, flumeview)
-          return ssb[name]
-        },
+      _flumeUse                : function (name, flumeview) {
+        ssb.use(name, flumeview)
+        return ssb[name]
+      },
 
       close                    : close,
-      del: valid.async(ssb.del, 'msgLink|feedId'),
+      del                      : valid.async(ssb.del, 'msgLink|feedId'),
       publish                  : valid.async(feed.add, 'string|msgContent'),
       add                      : valid.async(ssb.add, 'msg'),
-      queue                      : valid.async(ssb.queue, 'msg'),
+      queue                    : valid.async(ssb.queue, 'msg'),
```

Comment:
sorry, these misaligned dots have been eating my brain for 1-2 years.
I couldn't leave them alone